Rakefile

```diff
@@ -9,18 +9,39 @@ require "packaging/configuration"
 Packaging::Tasks.load_tasks(:exclude => ["tarball.rake"])
 
 require "yast/tasks"
-Yast::Tasks.submit_to(ENV.fetch("YAST_SUBMIT", "factory").to_sym)
+yast_submit = ENV.fetch("YAST_SUBMIT", "factory").to_sym
+Yast::Tasks.submit_to(yast_submit)
 
 Packaging.configuration do |conf|
   conf.package_name.sub!(/-.*/, "") # strip branch name
   conf.package_dir    = ".obsdir" # Makefile.ci puts it there
   conf.skip_license_check << /.*/
 
   # defined in Rakefile in https://github.com/openSUSE/packaging_rake_tasks
-  conf.obs_api = "https://api.opensuse.org/"
-  conf.obs_project = "filesystems:snapper"
-  conf.obs_target = "openSUSE_Factory"
-  conf.obs_sr_project = "filesystems:snapper"
+  if yast_submit == :sle12sp3
+    # IBS: Devel:YaST:Head/snapper
+    # see https://github.com/yast/yast-rake/blob/master/data/targets.yml
+    conf.obs_api = "https://api.suse.de/"
+    conf.obs_project= "Devel:YaST:Head"
+    conf.obs_sr_project= "SUSE:SLE-12-SP3:GA"
+    conf.obs_target = "sle12_sp3"
+  else
+    # OBS: filesystems:snapper/snapper
+    conf.obs_api = "https://api.opensuse.org/"
+    conf.obs_project = "filesystems:snapper"
+    conf.obs_sr_project = "filesystems:snapper"
+    conf.obs_target = "openSUSE_Factory"
+  end
+end
+
+desc 'Show configuration'
```

Comment:
Not as part of this PR, it would be nice to move this code to yast-rake. What do you think?